From 374294de467ebc115e70f0804e98081753d51c3b Mon Sep 17 00:00:00 2001
From: Thomas Huth <thuth@redhat.com>
Date: Thu, 19 Oct 2017 10:16:47 +0200
Subject: [PATCH 65/69] hw/dma/i8257: Remove redundant downstream
 user_creatable = false

RH-Author: Thomas Huth <thuth@redhat.com>
Message-id: <1508408209-5712-2-git-send-email-thuth@redhat.com>
Patchwork-id: 77372
O-Subject: [RHV7.5 qemu-kvm-rhev PATCH 1/3] hw/dma/i8257: Remove redundant downstream user_creatable = false
Bugzilla: 1503998
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Cornelia Huck <cohuck@redhat.com>
RH-Acked-by: Eduardo Habkost <ehabkost@redhat.com>

The i8257_class_init() function has already user_creatable = false
at the end of the function in the upstream code, so there is no
need to set this again with downstream-only code.

Signed-off-by: Thomas Huth <thuth@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hw/dma/i8257.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/hw/dma/i8257.c b/hw/dma/i8257.c
index e52a679..bd23e89 100644
--- a/hw/dma/i8257.c
+++ b/hw/dma/i8257.c
@@ -591,8 +591,6 @@ static void i8257_class_init(ObjectClass *klass, void *data)
     dc->reset = i8257_reset;
     dc->vmsd = &vmstate_i8257;
     dc->props = i8257_properties;
-    /* Disabled for Red Hat Enterprise Linux: */
-    dc->user_creatable = false;
 
     idc->get_transfer_mode = i8257_dma_get_transfer_mode;
     idc->has_autoinitialization = i8257_dma_has_autoinitialization;
-- 
1.8.3.1

